### PR TITLE
Nette 3.0 compatibility

### PR DESCRIPTION
**Pull request by Daaarkling**

### CODE REVIEW

**Comment by foxycode** [comment]
@davidkudera Looks like you don't have time for this package and I'm using it almost everywhere. I can continue maintaining it if you want to.

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-	"name": "carrooi/nette-menu",
+	"name": "darkling/nette-menu",
```
Comment (foxycode) [inline composer.json]:
```suggestion
	"name": "carrooi/nette-menu",
```

Why change name of package?

File: composer.json [hunk 2]
```diff
@@ -17,18 +17,19 @@
...
+		"nette/tester": "^2.1",
 		"mockery/mockery": "dev-master"
 	},
 	"autoload": {
 		"psr-0": {
 			"Carrooi\\Menu\\": "src/"
 		}
-	}
+	},
+	"minimum-stability": "beta"
```
Comment (foxycode) [inline composer.json]:
```suggestion
```
Shouldn't be needed any more.

File: tests/CarrooiTests/TestCase.php [hunk 3]
```diff
@@ -18,6 +18,7 @@
 use Nette\Application\UI\Presenter;
 use Nette\Http\Request;
 use Nette\Http\Url;
```
Comment (foxycode) [inline tests/CarrooiTests/TestCase.php]:
```suggestion
```

Could be removed.

File: composer.json [hunk 4]
```diff
@@ -17,18 +17,19 @@
...
-		"nette/tester": "^1.7",
+		"nette/tester": "^2.1",
 		"mockery/mockery": "dev-master"
 	},
 	"autoload": {
 		"psr-0": {
 			"Carrooi\\Menu\\": "src/"
 		}
-	}
+	},
```
Comment (foxycode) [inline composer.json]:
```suggestion
	}
```

**Comment by vojta001** [comment]
Ping @davidkudera: is there any chance this will get merged (possibly without the name change) or do we have to stick to forks?

File: tests/bootstrap.php [hunk 5]
```diff
@@ -7,6 +7,7 @@
 require_once __DIR__. '/CarrooiTests/TestCase.php';
 
 Tester\Environment::setup();
+Tester\Environment::bypassFinals();
```
Comment (davidkudera) [inline tests/bootstrap.php]: Why do we need this?

**Comment by davidkudera** [comment]
@vojta001 sure, I could merge it (without name change)

File: tests/bootstrap.php (hunk 5 above)
Comment (vojta001) [reply tests/bootstrap.php]: `Mockery\Exception: The class \Nette\Application\LinkGenerator is marked final and its methods cannot be replaced. Classes marked final can be passed in to \Mockery::mock() as instantiated objects to create a partial mock, but only if the mock is not subject to type hinting checks.`

**Comment by davidkudera** [comment]
@Daaarkling I'm closing this MR, solved in #38. Thanks for your work :+1:

**Pull request closed without merging**